{
  throw new UnsupportedOperationException("MockEnvironment does not support external task failure.");
}
